spec/integration/resource/api600/c7000/firmware_bundle/create_spec.rb

```diff
@@ -1,21 +0,0 @@
-# (C) Copyright 2017 Hewlett Packard Enterprise Development LP
```

Comment:
have you deleted this file intentionally?

Reply:
Yes. This check is done in V500.